### PR TITLE
Add my Crystagiri project

### PR DESCRIPTION
**Pull request by madeindjs**
Add my new project: [Crystagiri](https://github.com/madeindjs/Crystagiri) - An Html Reader / parser like [Nokogiri](https://github.com/sparklemotion/nokogiri) Ruby gem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -199,6 +199,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [C14N](https://github.com/vonKingsley/C14N) - Bindings for C14N (libxml2)
  * [crsfml](https://github.com/BlaXpirit/crsfml) - Bindings for [Simple and Fast Multimedia Library](http://www.sfml-dev.org/) (through CSFML)
  * [crt.cr](https://github.com/maiha/crt.cr) - Bindings for libncursesw and crt
+ * [crystagiri](https://github.com/madeindjs/Crystagiri) - An Html Reader / parser like [Nokogiri](https://github.com/sparklemotion/nokogiri) Ruby gem
```
Comment (veelenga) [inline README.md]: As far as i understand, this is not a low level binding, but a wrapper for Crystal's XML module. Let's move it to [Misc](https://github.com/veelenga/awesome-crystal#misc) section for now.

Comment (madeindjs) [inline README.md]: I updated my commit & move it in [misc](https://github.com/veelenga/awesome-crystal#misc)

Comment (madeindjs) [inline README.md]: I updated my commit & move it in [misc](https://github.com/veelenga/awesome-crystal#misc) section

**Comment by madeindjs** [comment]
I move it in [misc](https://github.com/veelenga/awesome-crystal#misc) section then I amend my commit and I `git push --force origin master`).

**Comment by veelenga** [comment]
Thank you @madeindjs

**Pull request merged**